docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -0,0 +1,255 @@
+ 
+    After:    
+  
+    ![DataGrid default link style after accessibility improvements](media/default-link-style-after.png)  
+
+For more information on WPF accessibility improvements in the .NET Framework 4.7.1, see [Accessibility improvements in WPF](../framework/migration-guide/retargeting/4.7-4.7.1.md#accessibility-improvements-in-wpf).
+
+## Windows Forms accessibility improvements
+
+In the .NET Framework 4.7.1, Windows Forms (WinForms) includes accessibility changes in the following areas.
+
+**Improved display in High Contrast mode**
+
+Starting with the .NET Framework 4.7.1, various WinForms controls offer improved rendering in the HighContrast modes available in the operating system. Windows 10 has changed the values for some high contrast system colors, and Windows Forms is based on the Windows 10 Win32 framework. For the best experience, run on the latest version of Windows and opt in to the latest OS changes by adding an app.manifest file in a test application and un-comment the Windows 10 supported OS  line so that it looks the following:
+
+```xml
+<!– Windows 10 –>
+<supportedOS Id=”{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}” />
+```
+Some examples of high contrast changes include:
+
+- Checkmarks in <xref:System.Windows.Forms.MenuStrip> items are easier to view.
+
+- When selected, disabled <xref:System.Windows.Forms.MenuStrip> items are easier to view.
+
+- Text in a selected <xref:System.Windows.Forms.Button> control contrasts with the selection color.
+
+- Disabled text is easier to read. For example:
+
+    Before:
+
+    ![Disabled text before accessibility improvements](media/wf-disabled-before.png) 
+
+    After:
+
+    ![Disabled text after accessibility improvements](media/wf-disabled-after.png) 
+- High contrast improvements in the Thread Exception Dialog.
+
+- Better multi-monitor DPI awareness in control anchoring and scaling.
+
+**Improved Narrator support**
+
+Windows Forms in the .NET Framework 4.7.1 includes the following accessibility improvements for the Narrator:
+
+- The <xref:System.Windows.Forms.MonthCalendar> control can be accessed by the Narrator, as well as by other UI automation tools.
+
+- The <xref:System.Windows.Forms.CheckedListBox> control notifies Narrator when the <xref:System.Windows.Forms.CheckedListBox.CheckedState> property has changed so the user is notified that they’ve changed the value of a list item.
+ 
+- The <xref:System.Windows.Forms.DataGridViewCell> control reports the correct read-only status to Narrator.
+ 
+- Narrator can now read disabled <xref:System.Windows.Forms.ToolStripMenuItem> text, whereas previously it would skip over disabled menu items.
+
+**Enhanced UI accessibility patterns**
+
+Starting with the .NET Framework 4.7.1, developers of accessibility technology tools can leverage common api accessibility patterns and properties for several WinForms controls. These accessibility improvements include:
```

Comment:
I thought that API is usually spelled in all-caps, no?

Reply:
it should be "Enhanced support for UIAutomation accessibility patterns" - this is how WPF calls UIA in the same doc

Reply:
Agree with @Tanya-Solyanik - "Enhanced support for UIAutomation accessibility patterns" sounds much better.